### ISSUE TITLE
Update accessibility docs regarding aria-hidden

### ISSUE THREAD

**Issue by scottaohara** [FortAwesome/Font-Awesome#14506]
As mentioned in [issue #13155](https://github.com/FortAwesome/Font-Awesome/pull/13155#issuecomment-454786217), font-awesome's [accessibility docs](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#web-fonts-decorative) currently use snippets like the following to showcase how to hide icons to screen readers:

```html
<i class="fas fa-camera-retro" aria-hidden></i>
```

These snippets should be updated to specifically call out `aria-hidden="true"`.  Without a set value, `aria-hidden` defaults to `undefined` and screen readers can still parse the content.

**Comment by robmadole** [comment]
@scottaohara do you have some examples of this behavior? Or at least which screen readers exhibit this?

**Comment by tagliala** [comment]
I don't know if there are actually screen readers exhibiting this, but as per specs

https://www.w3.org/TR/wai-aria-1.1/#aria-hidden


Characteristic | Value
-- | --
Used in roles|All elements of the base markup
Value|true/false/undefined


Value | Description
-- | --
false|The element is exposed to the accessibility API as if it was rendered.
true|The element is hidden from the accessibility API.
undefined|The element's hidden state is determined by the user agent based on whether it is rendered.

**Comment by scottaohara** [comment]
[example codepen](https://codepen.io/scottohara/pen/JwVKdj)
JAWS, NVDA and VoiceOver all announce the content of items where `aria-hidden` is set to the undefined value.

[ARIA 1.1 specification](https://www.w3.org/TR/wai-aria-1.1/#aria-hidden) notes:
>The element's hidden state is determined by the user agent based on whether it is rendered.

E.g. if it's rendered on screen, it's announced

**Comment by robmadole** [comment]
@scottaohara fantastic. thank you for sharing this! We'll get the docs updated.

**Comment by scottaohara** [comment]
thank you @robmadole
